.github/workflows/tests.yml

```diff
@@ -2,6 +2,8 @@ name: tests
 
 on:
   push:
+    branches:
```

Comment:
PRs and pushes to any branch were running together so we had duplicate jobs running when pushing to an open PR.

Reply:
Yeah, we nearly always work from forks in the OS repos.